### PR TITLE
New blog post for podman and bioinformatics

### PR DESCRIPTION
**Pull request by BryanHepworth**

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: _posts/2020-01-15-bioinformatics-with-rootless-podman.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+## By {{ author.display_name }} [GitHub](https://github.com/{{ author.github }}) [Twitter](https://twitter.com/{{ author.twitter }})
+
+## TL;DR
+
+Over the last 10 years I've seen machines and workflows evolve where I work. From the initial dedicated server, to hpc environments 
+and now the latest instance, containers.
+
+From an admin point of view this is great - The initial servers had to be carefully built and maintained so that everything would work nicely together. Incompatible programs at that time were run through a VM until such time as they could be folded in to the mix.
+
+The HPC's had versioned software and environment modules and were built to load the relevant dependencies at run time..
```
Comment (TomSweeneyRedHat) [inline _posts/2020-01-15-bioinformatics-with-rootless-podman.md]: did you mean to have a double ending period here?  "time.."

File: _posts/2020-01-15-bioinformatics-with-rootless-podman.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+Now we are into a new era, containers - and not just any old containers, but containers that end users can build and run up fairly 
+quickly to perform what-if's, and move on quickly through iterations until they perform the required functions.
+
+Podman has developed very rapidly and is incredibly easy to use. You can use it in conjuction with quay.io or run it on a local machine.
+
+I should add that Adrian Reber has also created a podman article using openhpc, well worth a read.https://youtu.be/TtHSNsbU24E
```
Comment (TomSweeneyRedHat) [inline _posts/2020-01-15-bioinformatics-with-rootless-podman.md]: Needs a space before the youtube link.  Also this is a video, which is great, kind of hard to read though.  Perhaps:

"I should add that Adrain Reber gave a [talk](https://youtu.be/TtHSNsbU24E) and has also created a Podman [article](https://podman.io/blogs/2019/09/26/podman-in-hpc.html) using openhpc; well worth a watch and a read."

File: _posts/2020-01-15-bioinformatics-with-rootless-podman.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+yum module install container-tools
+```
+RedHat 8.1 does rootless containers right out of the box. If you created a user during the setup, it'll have the details in /etc/subuid and /etc/subgid already.
+
+Log in with your userID and you can start creating a container
+```
+podman pull ubi8/ubi
+podman run --interactive --tty ubi8/ubi bash
+```
+The first command pulls down the ubi8 Universal Base Image, which is a great building block. The second command starts an interactive ubi8 image at a bash prompt. You can run any commands you like in this: -
```
Comment (TomSweeneyRedHat) [inline _posts/2020-01-15-bioinformatics-with-rootless-podman.md]: your call, and it's a style thing.  I pref "this:" over "this: -".  Not a biggy either way.

File: _posts/2020-01-15-bioinformatics-with-rootless-podman.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+localhost/whatshap-altered            latest   931b7f5302af   About a minute ago   545 MB
+localhost/whatshap                    latest   d523727fc6c2   3 days ago           545 MB
+registry.access.redhat.com/ubi8/ubi   latest   096cae65a207   5 weeks ago          239
+[nbh23@colombo ~]$
+```
+You can make multiple changes to your original container image until you are satisfied that it's working as you'd like.
+
+This has covered command line container image creation and usage, I'll be creating another blog post detailing graphical interactive containers as i'm aware that there are various interactive visual programs to cover too.
+
+feel free to contact me with any ideas or suggestions / questions
```
Comment (TomSweeneyRedHat) [inline _posts/2020-01-15-bioinformatics-with-rootless-podman.md]: "feel free" -> "Feel free"

File: _posts/2020-01-15-bioinformatics-with-rootless-podman.md [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+```
+[nbh23@colombo ~]$ podman run --interactive --tty ubi8/ubi bash
+[root@f471459c7619 /]# cat /etc/redhat-release
+Red Hat Enterprise Linux release 8.1 (Ootpa)
+[root@f471459c7619 /]#
+
+```
+Notice how the prompt changed from nbh23@colombo to root@f471459c7619 - the f471459c7619 is the part to remember, we'll interact with that later on in this post. It's a random allocation, so your instance will be different.
+
+The podman help menu's are excellent, podman -h gives you a list of subcommands, which you can then also query: -
```
Comment (TomSweeneyRedHat) [inline _posts/2020-01-15-bioinformatics-with-rootless-podman.md]: podman should be "Podman" except in actual commands.  "The Podman help..."

**Review by TomSweeneyRedHat — COMMENTED** [review]
A few nits, but looks really good overall Bryan, thanks a bunch for putting it together.

One other overall comment is this will end up showing only if you click on the "Blogs" button on the main menu.  If you want it to also show on the main menu you'll need to add a _posts/2020-01-15-new.md like this [one](https://github.com/containers/podman.io/commit/b6e7f9bd77f200b5a6eb226e4cdf0902777ee609).  If you don't want to mess with it, that's fine, I can add one after this merges.

**Review by BryanHepworth — COMMENTED** [review]
Alterations as suggested

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM, thanks so much for putting this all together Bryan!

**Pull request merged**